lib/ansible_tower_client/v2/job_template_v2.rb

```diff
@@ -1,5 +1,5 @@
 module AnsibleTowerClient
-  class JobTemplate < BaseModel
+  class JobTemplateV2 < JobTemplate
```

Comment:
Unfortunately the api itself isn't consistent here.  You hit `api/v1`and get the current api version.  There isn't an `api/v2` or even an `api/v3` yet.  We have to ask the config for the version and use that to decide which model version to load.  I was going to keep base models separate from version specific models and if we need to I'll create a v3 directory for any workarounds to the current version.